debezium-core/src/main/java/io/debezium/pipeline/txmetadata/TransactionContext.java

```diff
@@ -38,9 +38,13 @@
     private final Map<String, Long> perTableEventCount = new HashMap<>();
     private final Map<String, Long> viewPerTableEventCount = Collections.unmodifiableMap(perTableEventCount);
     private long totalEventCount = 0;
+    private long transactionStartTime = 0;
```

Comment:
I'd recommend to use an invalid value like `-1` to indicate unset value as `0` is theaoretically acceptable value.

Reply:
@jpechane Thanks for the review, I will address review comment this week.